### ISSUE TITLE
Access to the preprocessed dataset

### ISSUE THREAD

**Issue by mohitzsh** [balakg/voxelmorph#1]
Thanks for the nice work. 

I was wondering if you are planning to release the preprocessed code. I am trying to replicate the results in your paper. In the Dataset section, you mention that segmentations you received from the FreeSurfer were passed through a Quality Control step. It would be helpful if you can release the preprocessed dataset for public use. It would also be helpful if you can release the atlas you used in the standard .nii format so that I can align my own dataset to the atlas.

**Comment by ldf921** [comment]
I have the same issue. I am also working image registration with unsupervised learning. The preprocessed segmentation dataset would be very helpful for a fair comparison between our work and your work. Furthermore, I wonder if can also share the file list for the training set so that I can ensure I get all the training data.

**Comment by balakg** [comment]
Unfortunately, the data we used in our experiments have sharing regulations, so we can't provide them ourselves. You can also sign up to those studies listed in our paper and download the data if you'd like to. You could also try a dataset with provided segmentations such as the IXI dataset: http://brain-development.org/ixi-dataset/

As far as the atlas we used, it is located in data/atlas_norm.npz

**Comment by mohitzsh** [comment]
@balakg Thanks for the pointers. I'll try the IXI dataset. 

For the Atlas, it would be helpful if you can share in nifti format. I would like to use the NiftyReg software to affine register my training data to the atlas. If my understanding is correct, without the affine matrix, I think I can't write the atlas data from numpy array to .nii file.

**Comment by adalca** [comment]
@mohitsharma916, the atlas in the repo is what we use in training - it's been pre-processed, cleaned and cropped from the [original nifti](http://www.mit.edu/~adalca/proj/stroke/atlas/buckner61.nii.gz), whose construction is described in [1]. 

However, I'd recommend you use the numpy version available in the repo. If you'd like to make it into a nifti, you can essentially assume an identity affine transform and use nibabel, assuming you've loaded the atlas in a variable called `atlas_data`:
```
import nibabel as nib
img = nib.Nifti1Image(atlas_data, np.eye(4))
nib.save(img, '/.../atlas.nii')
```

[1] R. Sridharan‡, A.V. Dalca‡, K.M. Fitzpatrick, L. Cloonan, A. Kanakis, O. Wu, K.L. Furie, J. Rosand, N.S. Rost, P. Golland.  *Quantification and Analysis of Large Multimodal Clinical Image Studies: Application to Stroke.* In Proc. MICCAI International Workshop on Multimodal Brain Image Analysis (MBIA), pp. 18–30, 2013.

**Comment by mohitzsh** [comment]
@adalca Thanks for the comments. 

I think I can close the issue for now. I will reopen if something else comes up.

**Comment by mohitzsh** [comment]
I quick question about the preprocessing and automatic segmentation of the MRI volumes. Would it be possible to share the FreeSurfer functions you used for the preprocessing (normalization and segmentation)? I want to make sure that I can reproduce the results of the paper.

**Comment by adalca** [comment]
@mohitsharma916 we used FreeSurfer's [`recon-all`](https://surfer.nmr.mgh.harvard.edu/fswiki/recon-all). This might be be an overkill for what you want to do -- you can stop the process after the `aseg.mgz` is produced.

Note that segmentation (in this case via FS) is only necessary for validation of registration, not for the registration itself.

**Comment by Matt-arch** [comment]
hello there, I was working on an svm classifier which classifies the type of stroke for this project i am using the atlas dataset, the dataset have more than 200 stroke mri in the format of .nii, how can i load all of this .nii files in python using nibabel library

**Comment by Matt-arch** [comment]
hello there, I was working on an svm classifier which classifies the type of stroke for this project i am using the atlas dataset, the dataset have more than 200 stroke mri in the format of .nii, how can i load all of this .nii files in python using nibabel library

**Comment by adalca** [comment]
@Matt-arch please refer to #6 .
